pyscriptjs/src/remote_interpreter.ts

```diff
@@ -159,7 +158,7 @@ export class RemoteInterpreter extends Object {
         // for which the signature of `loadPackage` accepts the above params as args i.e.
         // the call uses `logger.info.bind(logger), logger.info.bind(logger)`.
         const messageCallback = logger.info.bind(logger) as typeof logger.info;
-        if (this.interpreter.version.startsWith('0.22')) {
+        if (this.interpreter.version.startsWith('0.22') || this.interpreter.version.startsWith('0.23')) {
```

Comment:
How about `Number(this.interpreter.version.split(".")[1]) >= 22`?

Reply:
Love it; changed to your version.